### PR TITLE
feat(perf): Landing page styling

### PR DESCRIPTION
**Pull request by udameli**
Double row widgets are now displayed before the triple row ones. Also updated triple row widget breakpoints so that the widgets are displayed in a single row for medium size viewports.

Fixes PERF-1830, PERF-1858

### CODE REVIEW

File: static/app/views/performance/landing/index.spec.tsx [hunk 1]
```diff
@@ -252,11 +252,11 @@ describe('Performance > Landing > Index', function () {
...
-    expect(titles.at(1)).toHaveTextContent('Transactions Per Minute');
-    expect(titles.at(2)).toHaveTextContent('Failure Rate');
-    expect(titles.at(3)).toHaveTextContent('Most Related Issues');
-    expect(titles.at(4)).toHaveTextContent('Most Improved');
+    expect(titles.at(0)).toHaveTextContent('Most Related Issues');
+    expect(titles.at(1)).toHaveTextContent('Most Improved');
+    expect(titles.at(2)).toHaveTextContent('User Misery');
+    expect(titles.at(3)).toHaveTextContent('Transactions Per Minute');
+    expect(titles.at(4)).toHaveTextContent('Failure Rate');
   });
```
Comment (udameli) [inline static/app/views/performance/landing/index.spec.tsx:260]: updating indices to reflect the row swap

**Review by DominikB2014 — APPROVED** [review]
Tested and lgtm!
Snapshot changes are expected.

**Comment by 0Calories** [comment]
In the 800px-1000px range the triple row widgets wrap, maybe we could display them in a single column instead?

image

**Review by 0Calories — APPROVED** [review]
noice 🎉

**Comment by udameli** [comment]
> In the 800px-1000px range the triple row widgets wrap, maybe we could display them in a single column instead?
> 
> image

I think it's expected   🤔 we want to make them legible while maintaining view hierarchy of top widget appearing larger than the bottom ones

**Pull request merged**